lib/services/runtime/handlers/noMatch.ts

```diff
@@ -10,20 +10,28 @@ export type NoMatchCounterStorage = number;
 
 export const EMPTY_AUDIO_STRING = '<audio src=""/>';
 
-const removeEmptyNoMatches = (noMatchArray?: string[]) => noMatchArray?.filter((noMatch) => noMatch != null && noMatch !== EMPTY_AUDIO_STRING);
+type NoMatchNode = BaseNode.Utils.BaseNode & VoiceNode.Utils.DeprecatedNodeNoMatch & { noMatch?: Nullable<VoiceNode.Utils.NodeNoMatch> };
+
+const removeEmptyNoMatches = (node: NoMatchNode) => {
+  const noMatches = node.noMatches || node.noMatch?.noMatches || [];
+  return noMatches?.filter((noMatch) => noMatch != null && noMatch !== EMPTY_AUDIO_STRING);
+};
+
+const getElseID = (node: NoMatchNode) => node.elseId ?? node.noMatch?.nodeID ?? null;
```

Comment:
Probably it's better to rename into `getNoMatchID` and use new data first:
```suggestion
const getNoMatchID = (node: VoiceNode.Utils.NoMatchNode) => node.noMatch?.nodeID ?? node.elseId ?? null;
```